client/odysseus/message/index.tsx

```diff
@@ -0,0 +1,74 @@
+import classnames from 'classnames';
+import { RefObject, useState } from 'react';
+import AsyncLoad from 'calypso/components/async-load';
+import type { Message } from '../types';
+
+import './style.scss';
+
+type ChatMessageProps = {
+	message: Message;
+	isLast: boolean;
+	messageEndRef: RefObject< HTMLDivElement >;
+};
+
+const ChatMessage = ( { message, isLast, messageEndRef }: ChatMessageProps ) => {
+	const [ liked, setLiked ] = useState( false );
+	const [ disliked, setDisliked ] = useState( false );
+
+	const isUser = message.role === 'user';
+
+	const onLike = () => {
+		if ( liked ) {
+			setLiked( false );
+			setDisliked( false );
+			return;
+		}
+		setLiked( true );
+		setDisliked( false );
+	};
+
+	const onDislike = () => {
+		if ( disliked ) {
+			setLiked( false );
+			setDisliked( false );
+			return;
+		}
+		setLiked( false );
+		setDisliked( true );
+	};
+
+	return (
+		<div
+			ref={ isLast ? messageEndRef : null }
+			className={ `chatbox-message ${ isUser ? 'user' : 'wapuu' }` }
+		>
+			<AsyncLoad require="react-markdown" children={ message.content } />
```

Comment:
You might want to set a `placeholder={ null }` if you don't mean to have the default placeholder here. 

Furthermore, is there a good reason to specify `children` as a separate prop rather than going with the standard React children syntax:

```
<AsyncLoad require="react-markdown">
	{ message.content }
</AsyncLoad>
```